### ISSUE TITLE
Get recent locations from the My prices page

### ISSUE THREAD

**Issue by odin-h** [openfoodfacts/open-prices-frontend#528]
### Problem
If you clear your browser cache, you lose your list of recent locations.

### Proposed solution
Check for previously added prices by the user and add their locations to localStorage if the recent locations list is empty.

**Comment by raphodn** [comment]
> if the recent locations list is empty

but maybe the user deleted the list on purpose ?

**Comment by odin-h** [comment]
Oh, you can delete entries from the recent locations list? I didn't realize. Maybe it would make more sense to call it favorite locations or saved locations or something instead? If the list doesn't always represent your recent locations?

**Comment by raphodn** [comment]
yes you can delete them individually, or all of them at once with the "Clear" button (in the location selector popup)

![image](https://github.com/openfoodfacts/open-prices-frontend/assets/7147385/889902e0-2836-4c2b-a550-94165f8c6449)

**Comment by odin-h** [comment]
I see. Well, I kinda still wish there was a way to populate the list with your actual recent locations from your price upload history so I don't have to memorize or look up the shop name again whenever I want to add a price from there again, as I sometimes switch devices or may clear my browser cache occasionally.
